### DISCUSSION TITLE
Jaka powinna być konwencja dotycząca struktury odnośników (linków).

### DISCUSSION THREAD

**Discussion by speplinski** [irdpl/wcag#6] [Organizacja pracy]
Skrypt, który generuje nazwy linków używa bezpośrednio tekstu, który jest podany, np. `po otrzymaniu fokusu` lub `spójna-nawigacja`.

To powoduje, że odnośniki mają polskie znaki `.../understanding/spójna-nawigacja.html`, a w konsekwencji problem z ich wywołaniem (gdyż nie wszystkie systemy potrafią poprawnie to interpretować).

```
function linkUnderstanding() {
	var understandingBaseURI;
	if (respecConfig.specStatus == "ED") understandingBaseURI = "../../understanding/";
	else understandingBaseURI = "https://www.w3.org/WAI/WCAG" + version + "/Understanding/";
	document.querySelectorAll('.sc,.guideline').forEach(function(node){
		var heading = textNoDescendant(findHeading(node));
		var pathFrag = titleToPathFrag(heading);
		if (node.id == "parsing") pathFrag = "parsing"; // special case parsing
		var el = document.createElement("div");
		el.setAttribute("class", "doclinks");
		el.innerHTML = "<a href=\"" + understandingBaseURI + pathFrag + ".html\">Objaśnienie " + heading + "</a> <span class=\"screenreader\">|</span> <br /><a href=\"https://www.w3.org/WAI/WCAG" + version + "/quickref/#" + pathFrag + "\">Jak spełnić " + heading + "</a>";
		if (node.className = "sc") node.insertBefore(el, node.children[2]);
		if (node.className = "guideline") node.insertBefore(el, node.children[1]);
	})
}
```

Czy nie powinno być tak, że jednak wykonujemy transliterację znaków polskich, w wyniku której zamieniamy znaki specjalne, takie jak ą, ć, ę, ł, ń, ó, ś, ż i ź, na ich odpowiedniki w standardowej alfabecie łacińskim?

**Comment by zwiastunsw** [comment]
Tak, wykonajmy transliterację. Później uważnie trzeba będzie nazywać pliki zgodnie z przyjętą konwencją zamiany znaków, żeby się linki nie pozrywały.
